wire-ios-request-strategy/Sources/APIs/MessageAPI.swift

```diff
@@ -62,8 +64,9 @@ class MessageAPIV0: MessageAPI {
     ) async throws -> (Payload.MessageSendingStatus, ZMTransportResponse) {
         let path = "/" + ["conversations", conversationID.uuid.transportString(), "otr", "messages"].joined(separator: "/")
 
+        // FIXME: [jacob] move encryption out of the API
```

Comment:
done in next PR?

Reply:
No, I don't plan on fixing this in this in the scope of this ticket.